search/result.hpp

```diff
@@ -148,6 +151,7 @@ class Result
   m2::PointD m_center;
   std::string m_str;
   std::string m_address;
+  uint32_t m_mainFeatureType = 0;
   uint32_t m_featureType = 0;
```

Comment:
Rename like m_mainType, m_matchedType?
m_matchedType can be 0.

Reply:
Done